### DISCUSSION TITLE
How do I skip windows as a Maui target in Uno Platform?

### DISCUSSION THREAD

**Discussion by KevKevin** [unoplatform/uno#15736] [Q&A]
Hi everyone,
when you implement Maui Embedding, the targets ios, android, maccatalyst and windows are "marked" as Maui.
In the template project there is this code:
![image](https://github.com/unoplatform/uno/assets/11044420/5748c835-dd91-482f-9c27-2b017c7a0da8)

With "maui" and "not_maui" you can switch between the different target and thats beautiful.

And when you target windows we can see the maui control.

Can we skip windows for being "maui"? Because I need to use maui for ios and android but use pure winui for Windows.

Thank you very much

K

**Comment by jeromelaban** [comment]
@dansiegel @eriklimakc I do not believe we're adding a `maui` namespace for the XAML conditionals. Is this documented somewhere as possible?

**Reply by dansiegel** [reply]
Well I'm pretty sure this is covered under the "best stuff" category which we may have been emulating Microsoft by not documenting at all... 🤦‍♂️ I've created an [issue](https://github.com/unoplatform/uno.extensions/issues/2206) to add this to the documentation.

@KevKevin if I understand your issue correctly you could make use of the `win` and `not_win` conditional namespaces to do something with MAUI but not on WinUI

```xml
<Page x:Class="UnoApp3.MainPage"
      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
      xmlns:maui="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
      xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
      xmlns:not_win="http://uno.ui/not_win"
      mc:Ignorable="d not_win"
      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
  <StackPanel
        HorizontalAlignment="Center"
        VerticalAlignment="Center">
    <maui:Grid>
      <win:TextBlock Text="This is Windows" />
      <not_win:TextBlock Text="This is not Windows" />
    </maui:Grid>
  </StackPanel>
</Page>
```

**Reply by RisingDragoon** [reply]
@dansiegel , yes my english isn't so good... Anyway, I need this configuration:
- a conditional namespace to use Android + ios (using maui embedding)
- a conditional namespace to use wasm + win

Because on mobile there is different layout than on win / wasm (for the size of screen).
I don't know if the `mobile` and `not_mobile` can resolve this. I saw the `maui` namespace thinking that it's needed to use maui embedding (?) or maybe not?

Thank you very much

**Reply by KevKevin** [reply]
Sorry I used my second account (I'm RisingDragoon)
